src/Sarif/ExtensionMethods.cs

```diff
@@ -153,15 +153,15 @@ public static void Populate(this Region region, NewLineIndex newLineIndex)
             }
         }
 
-        public static string GetMessageText(this Result result, IRule rule, bool concise = false)
```

Comment:
The purpose is to allow the _caller_ to pass in `consise = true;`. I'll come see you about this one.